### DISCUSSION TITLE
How to prevent CocSearch from openning new tabs after applying changes?

### DISCUSSION THREAD

**Discussion by zoumi** [neoclide/coc.nvim#4562] [Q&A] [answered]
before applying change there is only one tab:
![Snipaste_2023-03-11_16-10-08](https://user-images.githubusercontent.com/5162901/224473128-def0c129-153d-4398-bea5-26fd744ca685.jpg)

after applying changes there is three tabs:
![Snipaste_2023-03-11_16-10-26](https://user-images.githubusercontent.com/5162901/224473157-a3b4553f-1aec-4aa7-8be8-d3212f217545.jpg)

Env:
```
gvim 9
windows 10 64bit
```

**Comment by fannheyward** [comment, marked as answer]
`coc-refactor` will use `:h coc-config-workspace-openResourceCommand` to open files that related, by default `:h coc-config-workspace-openResourceCommand` is `tab drop`, you can change it in your coc-settings.json.

**Reply by zoumi** [reply]
I put this in :CocConfig, but it shows "Configuration property may not work as folder configuration (json)", why?

**Reply by zoumi** [reply]
This option works as expected. Thank you!

**Reply by NoahTheDuke** [reply]
I'm not sure which command to put in. Any suggestions?

**Reply by zoumi** [reply]
"workspace.openResourceCommand":"vsplit",
